linkml_model_enrichment/utils/enums_pvs_tsv.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import click
+import click_log
+import pandas as pd
+from linkml_runtime.utils.schemaview import SchemaView
+
+logger = logging.getLogger(__name__)
+click_log.basic_config(logger)
+
+
+@click.command()
+@click_log.simple_verbosity_option(logger)
+@click.option('--modelfile', help="path to LinkML input", type=click.Path(exists=True), show_default=True,
+              required=True)
+@click.option('--output_file', default="enums_pvs_tsv.tsv", help="path to TSV output", type=click.Path(),
```

Comment:
not good practice to include a default that relies on hardcoded files

Reply:
shoot, that's been a habit of mine. not a good practice even though it's just a filename, with no path?

I changed it anyway.